tests/test_patchanalysis.py

```diff
@@ -456,7 +456,7 @@ def attachmenthandler(attachments, bugid):
         self.assertEqual(info['landings']['aurora'], datetime(2016, 3, 28, 23, 24, 6, tzinfo=pytz.UTC))
         self.assertEqual(info['landings']['beta'], datetime(2016, 3, 31, 19, 48, 30, tzinfo=pytz.UTC))
         self.assertEqual(info['landings']['release'], datetime(2016, 4, 6, 19, 35, 18, tzinfo=pytz.UTC))
-        self.assertEqual(info['landings']['esr'], datetime(2016, 4, 7, 18, 41, 3, tzinfo=pytz.UTC))
+        self.assertIsNone(info['landings']['esr'])
```

Comment:
Why did this change?

Reply:
I think it's because i updated the mock versions file

Reply:
The `landings` property shouldn't be affected by the version: https://github.com/mozilla/libmozdata/blob/master/libmozdata/patchanalysis.py#L596.

Reply:
Fair enough, i'm looking into this

Reply:
@marco-c This is because i updated the mock versions file, now ESR is version 52.
As the `Bugzilla.get_landing_patterns` function uses the ESR version number to build the regex, the comment for esr45 does not match (see https://github.com/mozilla/libmozdata/blob/master/libmozdata/bugzilla.py#L304).